src/events/messageCreate.ts

```diff
@@ -48,7 +48,8 @@ export const run: RunFunction = async (bot, message: Message) => {
     triggers = triggers.concat(
       bot.functions.findTriggers(message.content.toLowerCase())
     );
-    const RegExp = /(https?:\/\/bin.ptdl.co\/([^\s]+))/g;
+    const RegExp =
+      /(https?:\/\/(bin.ptdl.co\/([^\s]+)|termbin.com\/([^\s]+)))/g;
```

Comment:
Termbin is not really known to me. Do we have to "index" it as well?

Reply:
There is no real difference between them. Both are user uploaded content, and that is about it. Just one is by pterodactyl and the other by `termbin`. I think it is the same "risk" level for both of them.

Reply:
There is no real difference between them. Both are user uploaded content, and that is about it. Just one is by pterodactyl and the other by termbin. I think it is the same "risk" level for both of them.